pegasus/migrations/79_migrate_properties_to_dynamodb.rb

```diff
@@ -0,0 +1,22 @@
+require 'cdo/hip_chat'
+require 'properties_api'
+
+Sequel.migration do
+  up do
+    HipChat.log 'Copying <b>key value pairs</b> to DynamoDB...'
+
+    # There are currently < 1000 rows in production. Don't bother batching.
+    rows = from(:app_properties)
+    rows.each do |row|
+      value = PropertyBag.parse_value(row[:value])
```

Comment:
Good catch Phil. It turns out there were 3 rows with bad JSON, at least one of which was due to  being truncated:
```
ubuntu@production-daemon:~/production$ bin/dashboard-console 
irb(main):035:0>     PEGASUS_DB[:app_properties].each do |row|
irb(main):036:1*       begin
irb(main):037:2*         value = PropertyBag.parse_value(row[:value])
irb(main):038:2>       rescue JSON::ParserError
irb(main):039:2>         puts "<b>Skipping bad key value pair with id #{row[:id]}</b>."
irb(main):040:2>         next
irb(main):041:2>       end
irb(main):042:1>     end
<b>Skipping bad key value pair with id 219</b>.
<b>Skipping bad key value pair with id 295</b>.
<b>Skipping bad key value pair with id 327</b>.
=> #<Sequel::Mysql2::Dataset: "SELECT * FROM `app_properties`">
```
I've updated the code to log this error and continue.